Fizz Buzz/README.markdown

```diff
@@ -55,39 +55,53 @@ Finding numbers divisible by five:
 
 ## The code
 
-Here is a simple implementation in Swift:
+Here is an implementation of the fizzBuzz algorithm using Swift 5 ".isMultiple" function with a Switch statement:
 
 ```swift
-func fizzBuzz(_ numberOfTurns: Int) {
-  for i in 1...numberOfTurns {
-    var result = ""
-
-    if i % 3 == 0 {
-      result += "Fizz"
-    }
-
-    if i % 5 == 0 {
-      result += (result.isEmpty ? "" : " ") + "Buzz"
+(0...50).forEach { (num) in
```

Comment:
I'm not a big fan of this. The range is now hard coded for 50 iterations, whereas the previous function allows the user to inject an arbitrary number of iterations.